src/features/lightspeed/explorerWebviewViewProvider.ts

```diff
@@ -71,25 +71,40 @@ export class LightspeedExplorerWebviewViewProvider
   private async _getWebviewContent(webview: Webview, extensionUri: Uri) {
     const userDetails =
       await this.lightspeedAuthenticatedUser.getLightspeedUserDetails(false);
-    if (userDetails) {
+    let markdownUserDetails =
+      await this.lightspeedAuthenticatedUser.getMarkdownLightspeedUserDetails(
+        false,
+      );
+    markdownUserDetails = String(markdownUserDetails);
+    console.log("==========markdownuserdetails==========");
+    console.log(markdownUserDetails);
+    console.log("=======================================");
+    let content = "";
+    if (markdownUserDetails !== "") {
+      content = markdownUserDetails;
+    } else if (userDetails) {
       const sessionInfo = getLoggedInUserDetails(userDetails);
       const userName = userDetails.displayNameWithUserType;
       const userType = sessionInfo.userInfo?.userType || "";
       const userRole =
         sessionInfo.userInfo?.role !== undefined
           ? sessionInfo.userInfo?.role
           : "";
-      return getWebviewContentWithActiveSession(
-        webview,
-        extensionUri,
-        userName,
-        userType,
-        userRole,
-        this.hasPlaybookOpened(),
-      );
+      content = `
```

Comment:
It would be better to handle the fallback scenario inside `getMarkdownLightspeedUserDetails()`.